test/functionalTest/cases/1308_subscription_cache/subCacheThrottlingAndExpirationAfterSubUpdate.test

```diff
@@ -0,0 +1,329 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Subscription Cache Throttling and Expiration After Sub-Update
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 38,101 IPv4 -multiservice -subCacheIval 5
+accumulatorStart
+
+--SHELL--
+
+#
+# 00. Create subscription for s0.* entities, attribute A, w/o tenant, expiration time 10 secs
+# 01. Update the subscription
+# 02. Create entity s01 w/ attr A w/o tenant
+# 03. Modify entity s01 w/ attr A w/o tenant - notification lost by throttling
+# 04. Modify entity s01 w/ attr A w/o tenant - notification lost by throttling
+# 05. Wait 5 secs so that the throttling of 4 secs is OK
```

Comment:
However, the payload shows that thottling is P1M (1 month), very small compared with 4 seconds. On the contrary, subscription duration is PT1S (1 second). Maybe both fields are incorrectly swapped...

If that's the case, I wonder how this .test is working... the difference between both situation is big.